espresso/client.go

```diff
@@ -0,0 +1,148 @@
+package espresso
```

Comment:
Eventually we should make this a standalone module

Reply:
https://github.com/EspressoSystems/espresso-sequencer/issues/731